contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -932,13 +929,10 @@ quota {
   ASSERT_NE(nullptr, method1);
 
   auto metric_cost_vector = method1->metric_cost_vector();
-  ASSERT_EQ(2, metric_cost_vector.size());
-  ASSERT_EQ("test.googleapis.com/operation/request",
-            metric_cost_vector[0].first);
-  ASSERT_EQ(10, metric_cost_vector[0].second);
+  ASSERT_EQ(1, metric_cost_vector.size());
```

Comment:
but we DO want to test 2 metrics case.   You can either convert the vector to a "set", then compare the "set".  or sort the vector first,